frontend/src/components/pages/GlobalCodeReviewDashboard/Presentation.jsx

```diff
@@ -119,6 +128,25 @@ export default function Presentation({
             onChange={handleChangeAssigneeTeamFilter}
           />
         </Paper>
+        <Typography variant="h6">Filter by card name</Typography>
+        <Paper>
+          <TextField
+            className={classes.filterByNameTextField}
+            variant="outlined"
+            placeholder="Card name"
+            value={cardNameFilterSearchTerm}
+            onChange={handleOnchangeCardNameFilter}
+            fullWidth
+          />
+          <FilterByNames
```

Comment:
Just a textbox is sufficient here.  It should work similarly to how the textboxes here work: https://tilde-front-dot-umuzi-prod.nw.r.appspot.com/users 

Or do you think that being able to toggle a bunch of things will be useful? It seems like a very heavy solution to me

Reply:
No, just a misunderstanding. I've updated it to only use a textbox.